### ISSUE TITLE
Bootstrap components are not working properly?

### ISSUE THREAD

**Issue by tharakaWijesundara** [LLK/scratch-gui#6361]
I tried to add **bootstrap button, dropdown menu, pop up window** etc. to stage selector window.(bottom right). But they are not working properly.**(styling issue**). What is the reason for that??

**Comment by apple502j** [comment]
Scratch 3.0 is React-based; why use Bootstrap?

**Comment by tharakaWijesundara** [comment]
Im using bootstrap for some development purpose. Why can't i use react-bootstrap?

**Comment by ghost** [comment]
Scratch does not use Bootstrap at all.

**Comment by tharakaWijesundara** [comment]
Yeah. I know it. The problem is why cant i use it by importing as node package??

**Comment by tharakaWijesundara** [comment]
So then, is there another way to add a dropdown menu?
